docs/conf.py

```diff
@@ -0,0 +1,48 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "outpack-py"
+copyright = "2024, Rich FitzJohn"
```

Comment:
Imperial holds our copyright; see LICENSE.txt; copy that over

Reply:
done